### PR TITLE
Signup: try business site setup flow

### PR DESCRIPTION
**Pull request by michaeldcain**
**This is an experimental branch - DO NOT MERGE**

This is a proof of concept flow to show how Headstart can be used to pre-populate a site with details provided by the user in Signup.

Requires D16829-code.

**To test:**
- Apply D16829-code and sandbox `public-api.wordpress.com`
- Visit calypso.localhost:3000/start/business-site-setup/

### CODE REVIEW

File: client/signup/steps/business-add-ons/index.jsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+						<FormLabel htmlFor="contactForm" className="business-add-ons__checkbox-label">
+							<FormInputCheckbox
+								name="contactForm"
+								id="contactForm"
+								className="business-add-ons__checkbox"
+								onChange={ this.handleChangeEvent }
+								value={ true }
+							/>
+							{ translate( 'A contact form' ) }
```
Comment (a8ci18n) [inline client/signup/steps/business-add-ons/index.jsx]: Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Contact Form' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/signup-business-site-setup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/theme-style/index.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+			<Button compact href="/jetpack/connect">
+				{ this.props.translate( 'Or Install Jetpack on a Self-Hosted Site' ) }
+			</Button>
+		);
+	}
+
+	render = () => {
+		const { translate } = this.props;
+		const headerText = translate( 'Choose a style for your website.' );
+		const subHeaderText = translate( "Don't worry, you can change this later." );
```
Comment (a8ci18n) [inline client/signup/steps/theme-style/index.jsx]: Hi! I've found a possible matching string that has already been translated **55** times:
`translate( 'Don't worry you can change this later.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by alisterscott** [comment]
@michaeldcain is this one still required?

**Comment by michaeldcain** [comment]
@alisterscott let's leave it open for now, while I write a wrap-up post about it.

**Pull request closed without merging**